### DISCUSSION TITLE
rtl support #81

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [GenieFramework/StippleUI.jl#83] [Ideas]
Quassar framework support right-to-left(RTL) for rtl cultures like arabic,hebr,persian...
Can we expect Stipple to support right to left as well?

**Comment by AbhimanyuAryan** [comment]
moved from here: https://github.com/GenieFramework/StippleUI.jl/issues/81

**Comment by essenciary** [comment]
@j209 I'm not familiar with this. What's needed to make it work? What have you tried and didn't work?
